### PR TITLE
Manage: post about adding a header for 'inactive' applications

### PR DESCRIPTION
**Pull request by MaeveRoseveare**
Added post: https://trello.com/c/v7qct3EC/1133-design-history-manage-adding-headers-for-providers-to-see-applications-submitted-over-30-days-ago

### CODE REVIEW

**Comment by MaeveRoseveare** [comment]
[Design history: Manage - adding headers for providers to see applications submitted over 30 days ago](https://trello.com/c/v7qct3EC/1133-design-history-manage-adding-headers-for-providers-to-see-applications-submitted-over-30-days-ago)

File: app/posts/manage-teacher-training-applications/2023-12-20-adding-heading-to-the-application-list.md [hunk 1]
```diff
@@ -0,0 +1,33 @@
+---
+title: Adding a heading to the application list to help providers find older applications
+description: We added an extra heading and content to help providers view applications submitted over 30 working days ago.
+date: 2023-11-30
```
Comment (lsalivonaite-transform) [inline app/posts/manage-teacher-training-applications/2023-12-20-adding-heading-to-the-application-list.md]:
```suggestion
date: 2023-12-20
```

File: app/posts/manage-teacher-training-applications/2023-12-20-adding-heading-to-the-application-list.md [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+
+We changed the colour of the text on applications submitted over 30 days ago. We decided to use colour and text so that we are not using colour alone to convey meaning which would cause issues with users who are colourblind or may perceive colour in different ways.
+
+![Screenshot of a list off applications with red text showing how long ago the application was received.](application-list.png)
+
+## Further considerations
+
+As a team, we are currently looking at more effective ways to help providers manage their workload, because headings and coloured text will not solve this issue entirely.
+
+We've also conducted a round of user research with training providers to understand their pain points with the increase in applications this cycle so we can look at designing solutions to help our users.
```
Comment (lsalivonaite-transform) [inline app/posts/manage-teacher-training-applications/2023-12-20-adding-heading-to-the-application-list.md]:
```suggestion
We have also conducted a round of user research with training providers to understand their pain points with the increase in applications this cycle so we can look at designing solutions to help our users.
```

I thought we couldn't use apostrophes/contractions, but i might be wrong

**Review by lsalivonaite-transform — CHANGES_REQUESTED** [review]

File: app/posts/manage-teacher-training-applications/2023-12-20-adding-heading-to-the-application-list.md (hunk 2 above)
Comment (MaeveRoseveare) [reply app/posts/manage-teacher-training-applications/2023-12-20-adding-heading-to-the-application-list.md]: We can sometimes, it's not a hard and fast rule. It's more negative contractions we should never use. But I think people have varying opinions of this. I thought the sentence was easier to read with the contraction, it rolled off the tongue better. lol.

Comment (lsalivonaite-transform) [reply app/posts/manage-teacher-training-applications/2023-12-20-adding-heading-to-the-application-list.md]: Ahaha, fair enough😁

**Review by lsalivonaite-transform — APPROVED** [review]

**Pull request merged**